### ISSUE TITLE
Mobile responsive or mobile app?

### ISSUE THREAD

**Issue by alvarofernandoms** [EspelhoPolitico/espelho-politico#141]
Today the application is not fully responsive. Let's use this issue for the discussion and for brainstorming the new layout. Is it interesting have a mobile app? What do you guys think about it?

**Comment by chocoelho** [comment]
First of all, mobile responsive and then native or hybrid app.

**Comment by chocoelho** [comment]
I think we should split back and front end. Back end can be used as Api

**Comment by alvarofernandoms** [comment]
I agree with the API idea. @danielteles and @matheusleite, what about a iOS application, hã? :smile:

**Comment by vital-edu** [comment]
I agree with @chocoelho, we should split the app in frontend + backend. In this case, I would recommend use Ionic. It is more easy to develop a multi-platform app (Windows Phone included).

But in this case, would be better do that right now, because when this happen almost everything will be thrown away

**Comment by matheusleite** [comment]
Use the backend as API is a great idea. 
Ionic is good and easy to develop. Ruby Motion ([rubymotion.com](url)) it's a cross-platform tool that uses Ruby.

The iOS Application is a good idea too, Swift is a great language and very easy and fast to learn. 
The native app advantage is that the design is slightly better compared to hybrid and has more power to use the device hardware ...
However, the iOS does not have the power so strong popularization as Android, and maybe this is very important to us.

**Comment by chocoelho** [comment]
If we use it as an API for the back end part I don't think storage or
parser will be a problem for a native app.

**Comment by vital-edu** [comment]
One huge problem: Both Ionic and RubyMotion are paid services. Because of that, I think that it would be better make a pure Android/iOS/WindowsPhone solution. But as @matheusleite said, we have to consider the impact of using these platforms against the effort needed. 

Considering that the EP seems to be more appropriate for web app than for mobile app application, we should just split it in backend and frontend and make a responsive application by now.
